tutorials/installation.md

```diff
@@ -188,13 +188,13 @@ The optional Eigen component of Gazebo Math requires:
 
   You can view available versions and their dependencies:
   ```
-  conda search libignition-math* --channel conda-forge --info
+  conda search libgz-math* --channel conda-forge --info
   ```
-  See the [Conda release repository](https://github.com/conda-forge/libignition-math4-feedstock) for more information.
+  See the [Conda release repository](https://github.com/conda-forge/libgz-math4-feedstock) for more information.
```

Comment:
this conda link doesn't work with the change. I'm not sure if any of the conda portion should be migrated actually

Reply:
Fixed! https://github.com/gazebosim/gz-math/pull/462/commits/bc118cbebbb21007d8e93acd9f0b5490c50baa30